sphericart-torch/src/autograd.cpp

```diff
@@ -205,7 +205,7 @@ torch::autograd::variable_list SphericalHarmonicsAutograd::forward(
             );
             printf("Re-attempting with GRID_DIM_X = 16\n");
 
-            calculator.CUDA_GRID_DIM_X_ = 16;
+            GRID_DIM_Y = 8;
```

Comment:
If we can change `GRID_DIM_Y`, we need to store it in the same way as we did GRID_DIM_X, to prevent spamming messages to the stdout

Reply:
We should also change the message above